inabox/tests/integration_v2_test.go

```diff
@@ -138,59 +106,88 @@ var _ = Describe("Inabox v2 Integration", func() {
 		Expect(err).Error()
 		Expect(err.Error()).To(ContainSubstring(getSolidityFunctionSig("ABNNotInFuture(uint32)")))
 
-		// ensure that a verifier can be added two blocks in the future
+		// ensure that a verifier can be added two blocks in the future where activation_block_number = latestBlock + 2
 		tx, err := eigenDACertVerifierRouter.AddCertVerifier(deployerTransactorOpts, uint32(latestBlock)+2, gethcommon.HexToAddress("0x0"))
 		Expect(err).To(BeNil())
-
 		mineAnvilBlocks(1)
-		latestBlock += 1
 
 		// ensure that tx successfully executed
-		receipt, err := ethClient.TransactionReceipt(ctx, tx.Hash())
+		err = validateTxReceipt(ctx, tx.Hash())
 		Expect(err).To(BeNil())
-		Expect(receipt).To(Not(BeNil()))
-		Expect(receipt.Status).To(Equal(uint64(1)))
 
 		// ensure that new verifier can be read from the contract at the future rbn
-		verifier, err := eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock+1))
+		verifier, err := eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock+2))
 		Expect(err).To(BeNil())
 		Expect(verifier).To(Equal(gethcommon.HexToAddress("0x0")))
 
 		// and that old one still lives at the latest block number - 1
-		verifier, err = eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock-1))
+		verifier, err = eigenDACertVerifierRouterCaller.GetCertVerifierAt(&bind.CallOpts{}, uint32(latestBlock-2))
 		Expect(err).To(BeNil())
 		Expect(verifier.String()).To(Equal(testConfig.EigenDA.CertVerifier))
 
-		// progress anvil chain to ensure latest block number is set for rbn so
-		// invalid verifier can be triggered
-		mineAnvilBlocks(3)
+		// progress anvil chain 10 blocks
+		mineAnvilBlocks(10)
 
-		// disperse blob #3 to trigger the new cert verifier
+		// disperse blob #3 to trigger the new cert verifier which should fail
+		// since the address is not a valid cert verifier and the GetQuorums call will fail
+		payload3 := randomPayload(1234)
+		cert3, err := payloadDisperser.SendPayload(ctx, payload3)
+		Expect(err.Error()).To(ContainSubstring("no contract code at given address"))
+		Expect(cert3).To(BeNil())
 
-		blob3, err := randomPayload(1000).ToBlob(codecs.PolynomialFormEval)
+		latestBlock, err = ethClient.BlockNumber(ctx)
 		Expect(err).To(BeNil())
 
-		reply3, err := disperseBlob(disp, blob3.Serialize())
+		// now disperse blob #4 to trigger the new cert verifier which should pass
```

Comment:
Is this comment misplaced? Doesn't seem to belong here

Reply:
[521de12](https://github.com/Layr-Labs/eigenda/pull/1643/commits/521de120b16396aab3f94f602757581721976319)